Maestro/subscriptions.json

```diff
@@ -91,6 +103,21 @@
             "-GitHubPullRequestNotifications 'dotnet/corefx-contrib'",
             "-DirPropsVersionElements ExternalExpectedPrerelease"
           ]
+        },
+        // This handler will bring the Latest ProjectKRel TFS release/1.0.0 build into CoreCLR release/1.0.0
+        {
+          "maestroAction": "coreclr-general-release",
+          "maestroDelay": "00:10:00",
+          "ScriptFileName": "UpdateDependencies.ps1",
+          "Arguments": [
+            "-GitHubUser dotnet-bot",
+            "-GitHubEmail dotnet-bot@microsoft.com",
+            "-GitHubPassword `$(`$Secrets['DotNetBotGitHubPassword'])",
+            "-VersionFileUrl https://github.com/dotnet/versions/blob/master/build-info/dotnet/projectk-tfs/release/1.0.0/Latest.txt",
+            "-GitHubUpstreamBranch 'release/1.0.0'",
+            "-GitHubPullRequestNotifications 'dotnet/coreclr-contrib; gkhanna79'",
```

Comment:
Yeah, this is a fallback so that once the group's created everyone's notified, but until then at minimum @gkhanna79 is notified. Easy to remove when the group is created. (I'll do that in this PR if timing works out that way.)